.travis.yml

```diff
@@ -3,18 +3,12 @@ before_install: gem update --system
 dist: trusty
 language: ruby
 rvm:
-- 2.0.0
```

Comment:
I have removed this, cause it tries to resolve **public_suffix** to **3.0.2**, which requires **Ruby version >= 2.1** 🔴 build [403456633](https://travis-ci.org/prontolabs/pronto/jobs/403456633)

Reply:
@dimitrovv could you make a separate PR for Travis changes? And let's figure out that one first.

Reply:
I saw you have already fixed this in [3b981f2](https://github.com/prontolabs/pronto/commit/3b981f2ecff5cba27ddb89ea7022d3c5fb1cfd99)